patterns/ecs-v1/postgresql

```diff
@@ -1,3 +1,3 @@
 # Default postgresql pg_log format pattern
-POSTGRESQL %{DATESTAMP:timestamp} %{TZ} %{DATA:user_id} %{GREEDYDATA:connection_id} %{POSINT:pid}
+POSTGRESQL %{DATESTAMP:timestamp} %{TZ:[event][timezone]} %{DATA:[user][name]} %{GREEDYDATA:[postgresql][log][connection_id]} %{POSINT:[process][pid]}
```

Comment:
was going to go with `postgresql.connection.id` 
but due better beats compatibility went with `postgresql.log.connection_id`

NOTE: that beat's `postgresql.log.core_id` is something different ...

Reply:
any objections going with beats compatibility (despite) the log parsing being different compared to beats?
// cc @webmat @ebeahan

Reply:
I have no objections.